.changeset/popular-wombats-promise.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/compiler': patch
```

Comment:
probably should be `major` I guess, right?

Reply:
idk, always have a hard time judging  these grey-area changes. Happy with whatever @natemoo-re thinks.